examples/dashboard.yml

```diff
@@ -0,0 +1,100 @@
+sources:
+  nyc_taxi:
+    description: Large version of nyc taxi dataset
+    driver: parquet
+    args:
+      urlpath: '{{ CATLOG_DIR }}data/nyc_taxi_wide.parq'
+      columns: ['dropoff_x','dropoff_y','pickup_x','pickup_y',
+                'dropoff_hour','pickup_hour','passenger_count']
+    metadata:
+      fields:
+        dropoff_hour:
+          label: Drop-off Hour
+        pickup_hour:
+          label: Pick-up Hour
+        fare_amount:
+          label: Fare
+          unit: $
+        passenger_count:
+          label: Passenger Count
+        counts:
+          label: Ride counts
+      plot:
+        xlim: !!python/tuple [-8240227.037, -8231283.905]
+        ylim: !!python/tuple [4974203.152, 4979238.441]
+        kind: points
+      plots:
+        dropoff:
+          label: NYC Taxi Dropoffs
+          x: dropoff_x
+          y: dropoff_y
+        pickup:
+          label: NYC Taxi Pickups
+          x: pickup_x
+          y: pickup_y
+          
+  census:
+    description: US Census Synthetic Data
+    driver: parquet
+    args:
+      urlpath: '{{ CATLOG_DIR }}data/census.snappy.parq'
+    metadata:
+      fields:
+        race:
+          label: Race
+# this doesn't work, but I think something like it might. 
+#           labels:
+#             w: White
+#             b: Black
+#             a: Asian
+#             h: Hispanic
+#             o: Other
+        counts:
+          label: Counts
+      plot:
+        x: easting
+        y: northing
+        xlim: !!python/tuple [-13914936, -7235767]
+        ylim: !!python/tuple [2632019, 6446276]
+        kind: points  
+      plots:
+        people:
+          label: US Census Synthetic people
+        race:
+          label: US Census Synthetic race
+          c: race
+          cmap:
+            w: blue
+            b: green
+            a: red
+            h: orange
+            o: saddlebrown
+  
+  opensky:
+    description: OpenSky Flight Paths
+    driver: netcdf
+    args:
+      urlpath: '{{ CATLOG_DIR }}data/opensky.h5'
```

Comment:
Is ``{{ CATLOG_DIR }}`` really the correct way to spell this? I've always used ``{{ CATALOG_DIR }}/``?

Reply:
I think it looks more natural with the /, but it seems like both ways work?

Reply:
Oh, you mean CAT LOG?  That seems like a typo.

Reply:
This one hasn't been tested, so I'll just go ahead and fix that.

Reply:
I meant both things, glad it's a typo though.

Reply:
>This one hasn't been tested, so I'll just go ahead and fix that.

The typo appears in all of them.

Reply:
Odd; not sure how it's ever worked, then. "CATLOG" doesn't appear in the intake docs or repo.

Reply:
(Scratches head) Odd.  I've fixed that and pushed.

Reply:
The variable was probably just replaced with an empty string so it would work as long as the dashboard.ipynb is in the same place as the data directory.